src/components/Checkbox.js

```diff
@@ -13,27 +13,33 @@ const propTypes = {
     onPress: PropTypes.func.isRequired,
 
     /** Should the input be styled for errors  */
-    hasError: PropTypes.bool,
+    errorText: PropTypes.string,
 
     /** Should the input be disabled  */
     disabled: PropTypes.bool,
+
+    /** A ref to forward to the Pressable */
+    forwardedRef: PropTypes.oneOfType([PropTypes.object]),
```

Comment:
This type is wrong.

Reply:
Should I do it this way and keep the default value as undefined

 ```js 
forwardedRef: PropTypes.oneOfType([
            PropTypes.func,
            PropTypes.shape({current: PropTypes.instanceOf(React.Component)}),
        ]),
```

Reply:
@thesahindia can it ever be `PropTypes.instanceOf(React.Component)`?

Reply:
@thesahindia are you sure it can be `PropTypes.instanceOf(React.Component)`?

Reply:
@rushatgabhane, It's being used in codebase, let me know if something else should be used.

Reply:
I believe `PropTypes.func` is more accurate.

Reply:
> I believe `PropTypes.func` is more accurate.

Getting error

<img width="383" alt="Screenshot 2022-02-28 at 11 56 43 PM" src="https://user-images.githubusercontent.com/83179295/156037875-bb8c2db4-0eef-4569-aba0-80ca6b7c8912.png">

Reply:
@thesahindia Thanks, let's go ahead with [this](https://github.com/Expensify/App/pull/7750#discussion_r814288066) then.